### PR TITLE
Split some words from template to improve translation.

### PR DESCRIPTION
**Pull request by L0veSunshine**
As title.

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1380,8 +1380,13 @@ settings.search_user_placeholder = Search user…
 settings.org_not_allowed_to_be_collaborator = Organizations cannot be added as a collaborator.
 settings.change_team_access_not_allowed = Changing team access for repository has been restricted to organization owner
 settings.team_not_in_organization = The team is not in the same organization as the repository
+settings.teams = Teams
+settings.add_team = Add Team
 settings.add_team_duplicate = Team already has the repository
 settings.add_team_success = The team now have access to the repository.
+settings.search_team = Search Team...
```
Comment (lafriks) [inline options/locale/locale_en-US.ini]:
```suggestion
settings.search_team = Search Team…
```

**Review by lafriks — DISMISSED** [review]

File: templates/repo/settings/collaboration.tmpl [hunk 2]
```diff
@@ -66,7 +66,7 @@
 							{{.Name}}
 						</a>
 					</div>
-					<div class="ui eight wide column poping up" data-content="Team's permission is set on the team setting page and can't be changed per repository">
+					<div class="ui eight wide column poping up" data-content={{$.i18n.Tr "repo.settings.change_team_permission_tip"}}>
```
Comment (6543) [inline templates/repo/settings/collaboration.tmpl]:
```suggestion
					<div class="ui eight wide column poping up" data-content="{{$.i18n.Tr "repo.settings.change_team_permission_tip"}}">
```

File: templates/repo/settings/collaboration.tmpl [hunk 3]
```diff
@@ -79,7 +79,7 @@
 						{{end}}
 					</div>
 					{{if $allowedToChangeTeams}}
-						<div class="ui two wide column {{if .IncludesAllRepositories}}poping up{{end}}" {{if .IncludesAllRepositories}}data-content="This team has access to all repositories and can't be removed."{{end}}>
+						<div class="ui two wide column {{if .IncludesAllRepositories}}poping up{{end}}" {{if .IncludesAllRepositories}} data-content= {{$.i18n.Tr "repo.settings.delete_team_tip"}} {{end}}>
```
Comment (6543) [inline templates/repo/settings/collaboration.tmpl]:
```suggestion
						<div class="ui two wide column {{if .IncludesAllRepositories}}poping up{{end}}" {{if .IncludesAllRepositories}} data-content="{{$.i18n.Tr "repo.settings.delete_team_tip"}}"{{end}}>
```

File: templates/repo/settings/collaboration.tmpl [hunk 4]
```diff
@@ -96,15 +96,15 @@
 					<div class="inline field ui left">
 						<div id="search-team-box" class="ui search" data-org="{{.OrgName}}">
 							<div class="ui input">
-								<input class="prompt" name="team" placeholder="Search teams..." autocomplete="off" autofocus required>
+								<input class="prompt" name="team" placeholder={{$.i18n.Tr "repo.settings.search_team"}} autocomplete="off" autofocus required>
```
Comment (6543) [inline templates/repo/settings/collaboration.tmpl]:
```suggestion
								<input class="prompt" name="team" placeholder="{{$.i18n.Tr "repo.settings.search_team"}}" autocomplete="off" autofocus required>
```

**Review by 6543 — CHANGES_REQUESTED** [review]

**Review by 6543 — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
@lafriks this needs your review

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**